### DISCUSSION TITLE
ragflow-server" reports "HTTP 500" error

### DISCUSSION THREAD

**Discussion by cjcmtcjcmt** [infiniflow/ragflow#2234] [Q&A]
Running "ragflow-server" on Docker in Windows 10 reports "HTTP 500" error. Do you have any suggested solutions?
![ragflow-serve 500 error](https://github.com/user-attachments/assets/6b740f9c-6c2e-40e3-b550-0ab4ee2c18d8)

**Comment by KevinHuSh** [comment]
No clue yet.
